charts/hlf-k8s/README.md

```diff
@@ -74,16 +78,17 @@ The following table lists the configurable parameters of the hlf-k8s chart and d
 | `systemChannel.name` | The name of the system channel | `systemchannel` |
 | `systemChannel.organizations` | The organizations to add to the system channel. See [Add an organization to the system channel](#add-an-organization-to-the-system-channel). | `[]` |
 | **Common / Other** |  |  |
-| `image.repository` | `hlf-k8s` image repository | `substrafoundation/hlf-k8s` |
-| `image.tag` | `hlf-k8s` image tag | `latest` |
-| `image.pullPolicy` | Image pull policy | `IfNotPresent` |
+| `images.hlfk8s.repository` | `hlf-k8s` image repository | `substrafoundation/hlf-k8s` |
+| `images.hlfk8s.tag` | `hlf-k8s` image tag | `latest` |
+| `images.hlfk8s.pullPolicy` | Image pull policy | `IfNotPresent` |
+| `images.hlfk8sCaEnrollment.repository` | `hlf-k8s-ca` image repository | `substrafoundation/hlf-k8s-ca` |
+| `images.hlfk8sCaEnrollment.tag` | `hlf-k8s-ca` image tag | `latest` |
+| `images.hlfk8sCaEnrollment.pullPolicy` | Image pull policy | `IfNotPresent` |
```

Comment:
I would name that fabric-tools.image and fabric-ca-tools.image (at least put the "image" as a subpath of the service)